.conda/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set pyproject = load_file_data('../pyproject.toml', from_recipe_dir=True) %}
+{% set project = pyproject.get('project') %}
+{% set urls = pyproject.get('project', {}).get('urls') %}
+{% set version = environ.get('BUILD_VERSION', '0.8.0a0') %}
+package:
+  name: {{ project.get('name') }}
+  version: {{ version }}
+
+source:
+  fn: {{ project.get('name') }}-{{ version }}.tar.gz
+  url: ../dist/{{ project.get('name') }}-{{ version }}.tar.gz
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python>=3.8, <4.0
+    - setuptools
+
+  run:
+    - importlib_metadata
+    - numpy >=1.16.0, <2.0.0
+    - scipy >=1.4.0, <2.0.0
+    - pillow >=9.2.0
+    - matplotlib >=3.1.0
+    - h5py >=3.1.0, <4.0.0
+    - opencv >=4.5.0, <5.0.0
+    - pypdfium2_helpers >=4.0.0, <5.0.0
```

Comment:
`pypdfium2-team::pypdfium2_helpers` ?

Reply:
Yes, but I believe it ties the package to the right channel.
e.g. if a pypdfium2_helpers package appeared in some other higher-priority channel, the above should presumably override that